datasets/management/commands/searchvector.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.postgres.search import SearchVector
+from django.core.management.base import BaseCommand
+
+from datasets.models import Gazette
+
+
+class Command(BaseCommand):
+    help = """Remonta os indices de busca e diários em caso de problemas
+            com a geração de índice via trigger"""
+
+    def echo(self, text, style=None):
+        self.stdout.write(style(text) if style else text)
+
+    def handle(self, *args, **options):
+        gazette_count = Gazette.objects.count()
+        self.echo(
+            f"Creating search vector for Gazette. Total items: {gazette_count}",
```

Comment:
A total minor improvement for a better readability: adding thousands separator:

```suggestion
            f"Creating search vector for Gazette. Total items: {gazette_count:,}",
```